docs/mine/lotus/miner-lifecycle.md

```diff
@@ -0,0 +1,125 @@
+
+Miner: t01001
+Current Epoch:           500
+Proving Period Boundary: 154
+Proving Period Start:    154 (2h53m0s ago)
+Next Period Start:       3034 (in 21h7m0s)
+Faults:      768 (100.00%)
+Recovering:  768
+Deadline Index:       5
+Deadline Sectors:     0
+Deadline Open:        454 (23m0s ago)
+Deadline Close:       514 (in 7m0s)
+Deadline Challenge:   434 (33m0s ago)
+Deadline FaultCutoff: 384 (58m0s ago)
+```
+
+In this next example, the miner can be safely restarted because no `Deadlines` are earlier than `Current Epoch`. You have about 45 minutes before the miner must be back online to declare faults. This is known as the `Deadline FaultCutoff`. If the miner has no faults, you have about an hour.
+
+```bash
+$ sudo lotus-miner proving info
+
+Miner: t01000
+Current Epoch:           497
+Proving Period Boundary: 658
+Proving Period Start:    658 (in 1h20m30s)
+Next Period Start:       3538 (in 25h20m30s)
+Faults:      0 (0.00%)
+Recovering:  0
+Deadline Index:       0
+Deadline Sectors:     768
+Deadline Open:        658 (in 1h20m30s)
+Deadline Close:       718 (in 1h50m30s)
+Deadline Challenge:   638 (in 1h10m30s)
+Deadline FaultCutoff: 588 (in 45m30s)
+```
+
+Once you have verified that your miner is safe to shut down, you can run `lotus-miner stop`. If you are using systemd, run `systemctl stop lotus-miner`.
+
+You can restart the miner as soon as you wish. Any sealing operations that were ongoing will be reset to the last checkpoint and continue.
+
+Workers do not need to be restarted as they will reconnect to the miner automatically when it comes back up. However, if you are upgrading your node at the same time as shutting down your miner, you will need to restart the machine.
+
+## Restarting workers
+
+Lotus [seal workers](seal-workers.md) can be restarted any time, but if they are in the middle of one of the sealing steps, then the operation will start again (from the last checkpoint).
+
+::: warning
+There is a maximum of three attempts to complete a sealing operation before it is fully started from scratch (_pre-commit1_ phase).
+:::
+
+## Changing storage locations
+
+If you wish to change the location of your miner-related storage to a different path, for the miner or the seal workers, make sure that the Lotus miner and any seal workers are aware of the new location.
+
+```sh
+lotus-miner storage list
+```
+
+The above command will give you an overview of [storage locations known to the miner](custom-storage-layout.md). This information is stored in `~/lotusminer/storage.json` (or `$LOTUS_MINER_PATH/storage.json` if defined). Lotus seal workers store all the data in the `~/lotusworker` folder (or `$LOTUS_WORKER_PATH` if defined).
+
+If you wish to change any of the storage locations of the Lotus miner, follow these steps:
+
+1. [Stop the miner](#safely-restarting-the-miner-daemon)
+2. Copy or move the data. You should move the data folder from the original location to the destination without changing any of the content.
```

Comment:
```suggestion
1. Copy or move the data. You should move the data folder from the original location to the destination without changing any of the content.
2. [Stop the miner](#safely-restarting-the-miner-daemon)
```
Stopping the miner first will result in definitely missing many window posts purely due to the amount of time it will take to transfer any non-trivial number of sectors. i.e. 1GbE LAN takes just over ~1hr to transfer 448GiBs (or 14 sectors).

Reply:
I'm guessing the data should not be moved then, while the miner is running, only copied. And I am guessing that even if the miner is running, it should not take on any new deals etc while the copy is ongoing?

Reply:
> I'm guessing the data should not be moved then, while the miner is running, only copied

correct
> it should not take on any new deals etc while the copy is ongoing?

correct, also don't start pledging any new sectors, obviously.